feathr_project/feathr/api/app/core/configs.py

```diff
@@ -0,0 +1,13 @@
+import logging
+import os
+
+API_Version = "/v0.1"
+log_level = os.getenv("logLevel", "INFO")
+app_insights_connection_string = os.getenv("APPLICATIONINSIGHTS_CONNECTION_STRING")
```

Comment:
document it to make sure folks know how to get it

Reply:
OK, will update the design doc.